horizon/enforcer/schemas_kong.py

```diff
@@ -76,5 +76,11 @@ class KongAuthorizationQuery(BaseSchema):
     input: KongAuthorizationInput
 
 
+class KongWrappedAuthorizationQuery(BaseSchema):
+    user: Dict
```

Comment:
Why not describe the full schema with extra=ignore?

Reply:
the is_allowed function uses dict on the model provided, the other schemas have user of type User